app/forms/validators.py

```diff
@@ -445,7 +454,7 @@ def _is_valid(
 
 def format_playback_value(
     value: Union[float, Decimal], currency: Optional[str] = None
-) -> str:
+) -> Any:
```

Comment:
This should return a str, as we are enforcing the var being returned as a string.

Reply:
This needed to be any because the return on 459 returns `get_formatted_currency` which needs to return Any because it uses a function from babel that returns Any